ultra/ultra/train.py

```diff
@@ -45,24 +45,77 @@
 num_gpus = 1 if torch.cuda.is_available() else 0
 
 
-def train(
-    scenario_info,
-    num_episodes,
-    policy_classes,
-    max_episode_steps,
-    max_steps,
-    eval_info,
-    timestep_sec,
-    headless,
-    seed,
-    log_dir,
-    policy_ids=None,
-):
-    torch.set_num_threads(1)
-    total_step = 0
-    finished = False
-    evaluation_task_ids = dict()
+def create_parser():
```

Comment:
Perhaps just for clarity?
```suggestion
def create_argument_parser():
```
I noticed in SMARTS they have something similar here: https://github.com/huawei-noah/SMARTS/blob/master/examples/__init__.py

If we go with this change, then of course the calls to this function should be changed as well.